interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -18,6 +18,7 @@
 #include "cling/Interpreter/RuntimeOptions.h"
 
 #include "llvm/ADT/StringRef.h"
+#include "llvm/ExecutionEngine/Orc/LLJIT.h"
```

Comment:
We don't need to include the header, just forward-declaring `llvm::orc::LLJIT` should be sufficient to return a pointer to it.